src/actions/MenuActions.js

```diff
@@ -3,6 +3,9 @@ import axios from 'axios';
 
 // REQUESTS
 
+let breadcrumbsCache = {};
```

Comment:
@teosarca Removed the cache.

All things you told above are correct and valid. But this particular piece simply didn't get updated that often. That's why I decided to reduce requests.

Now it removed and just don't fire duplicate requests sent for the same breadcrumb items.